src/init.cpp

```diff
@@ -383,6 +383,10 @@ std::string HelpMessage(HelpMessageMode mode)
     strUsage += HelpMessageOpt("-walletnotify=<cmd>", _("Execute command when a wallet transaction changes (%s in cmd is replaced by TxID)"));
     strUsage += HelpMessageOpt("-zapwallettxes=<mode>", _("Delete all wallet transactions and only recover those parts of the blockchain through -rescan on startup") +
         " " + _("(1 = keep tx meta data e.g. account owner and payment request information, 2 = drop tx meta data)"));
+<<<<<<< HEAD
+=======
+
+>>>>>>> Removes out bitcoin mention in favor for zcash
```

Comment:
Ugh thought I got rid of this. Deleting. 

Makes sense to pull in bitcoin/bitcoin#6504 first though